sphinx/developers/components.rst

```diff
@@ -183,6 +167,19 @@ Decoupled OME data model components:
 - :ref:`OME-XML <ome-xml>`
 - :ref:`Specification <specification>`
 
+.. _bio-formats-examples:
+
+`Bio-Formats examples <https://github.com/ome/bio-formats-examples>`_
+
+Usage examples for using Bio-Formats with the Maven or Gradle build
```

Comment:
Minor nitpick but do we need both "usage" and "using" in this sentence? Just "Examples for using..." seems like it would be fine